models/unittest/testdb.go

```diff
@@ -39,19 +39,27 @@ func fatalTestError(fmtStr string, args ...interface{}) {
 	os.Exit(1)
 }
 
+// TestOptions represents test options
+type TestOptions struct {
+	GiteaRootPath string
+	FixtureFiles  []string
+	TearUp        func() error
```

Comment:
I think what you meant is `Setup`, not `TearUp`

Reply:
Done.